backend/core/README.md

```diff
@@ -8,22 +8,47 @@ OpenAPI (fka Swagger) documentation is automatically generated by the server at
 
 ## Getting Started for Developers
 
-The following steps should provide a working local environement for development purposes:
+The following steps should provide a working local environment for development purposes:
 
 ### Setting up your local environment variables
 
 Operational configuration the service is read from environment variables. Copy `.env.template` to `.env` and edit the settings specific to your development environment. Make sure the Database URL and Test Database URL match your Postgres configuration.
 
+### Install Node
+
+Install Node.js 14.x
+
+`brew install node@14.`
+
 ### Installing Postgres
 
-You can install Postgres using Homebrew with the following command: `brew install postgresql`.
+You can install Postgres 12 using Homebrew with the following command: `brew install postgresql`.
```

Comment:
I think I recall having to do another step like setting up a user, but we can update this if Yazeed runs into any issues.